ui-v2/app/templates/dc/acls/policies/-notifications.hbs

```diff
@@ -0,0 +1,20 @@
+{{#if (eq type 'create')}}
```

Comment:
Though the if / else party here is a bit iffy (pun intended :D) - this will be nice when we have to translate the UIs.

Reply:
Oh I know, it's lovely isn't it 😄 . I don't think there are any `switch`es in `htmlbars` - I've been trying to keep all the text content in the templates, although I'm mulling that over again 🤔